### PR TITLE
test(examples): expand to also run on arm64 machine

### PR DESCRIPTION
**Pull request by AtomicFS**
- add example test for `coreboot` on `arm64`
- add example test for `uroot` on `arm64`
- add example test for `linux` on `arm64`
- add example test for `stitching` on `arm64`

edk2 will come in later PR

### CODE REVIEW

**Comment by AtomicFS** [comment]
The issue is failing because of missing git submodule, which is fixed in #392 , so should be fine too @MDr164

**Comment by AtomicFS** [comment]
There was some left-over garbage in `arm64` runner which was causing problems. Is fixed now.

File: .github/workflows/example.yml [hunk 1]
```diff
@@ -89,16 +89,22 @@ jobs:
   # Example of building coreboot
   # ANCHOR: example_build_coreboot
   build-coreboot:
-    runs-on: ubuntu-latest
```
Comment (MDr164) [inline .github/workflows/example.yml:92]: Could you briefly explain why this got removed? Otherwise the PR looks fine to me.

Comment (AtomicFS) [reply .github/workflows/example.yml:92]: Yes, it is replaced with
```yaml
runs-on: ${{ matrix.arch == 'arm64' && 'ARM64' || 'ubuntu-latest' }}
```

So basically it picks whether to run on `ubuntu-latest` or our self-hosted runner.

Comment (MDr164) [reply .github/workflows/example.yml:92]: Oh I missed that line, thanks for the explanation :+1:

**Review by MDr164 — DISMISSED** [review]

**Comment by AtomicFS** [comment]
Ups, the artifacts had conflicting names. Fixed.

**Review by MDr164 — APPROVED** [review]

**Pull request merged**